src/util/sss_ssh.c

```diff
@@ -18,13 +18,59 @@
     along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
+#include <ctype.h>
 #include <arpa/inet.h>
 
 #include "db/sysdb.h"
 #include "util/util.h"
 #include "util/crypto/sss_crypto.h"
 #include "util/sss_ssh.h"
 
+
+#define SKIP_SPACES(p)       while (*p != '\0' &&  isspace(*p)) { p++; }
+#define SKIP_UNTIL_SPACE(p)  while (*p != '\0' && !isspace(*p)) { p++; }
```

Comment:
Does it really need to check for "space, form-feed  ('\f'),  newline  ('\n'),  carriage  return ('\r'), horizontal tab ('\t'), and vertical tab ('\v')" or `strchr(' ')` could be used instead?

Reply:
Actually no. I though the file could have TABs, but it seems it can only have spaces. I will check for ' '.